pkgs/top-level/all-packages.nix

```
@@ -1375,6 +1375,8 @@ with pkgs;
 
   substituteAllFiles = callPackage ../build-support/substitute-files/substitute-all-files.nix { };
 
+  substituteAttrs = callPackage ../build-support/substitute/substitute-attrs.nix { };
```

Do you think in this case it'd be good to make a start with [ending a homonym](https://github.com/NixOS/nixpkgs/issues/237216#issuecomment-2045821891)?

Something like:

```suggestion
  replaceByAttrs = callPackage ../build-support/substitute/substitute-attrs.nix { };
```

---

I like it!

---

I decided to go with `substituteInPath` per Silvan's suggestion, but I _do_ like `replaceByAttrs` and could easily be swayed to that name.

---

How about `replaceInPath`?

> This isn't [Nix substitution](https://nixos.org/manual/nix/stable/glossary#gloss-substituter)

---

Updated.